### PR TITLE
feat: post refactoring invoice ui

### PR DESCRIPTION
**Pull request by JustSamuel**
# Description
Now the UI adheres to the new setup: the user is able to change the amount of the invoice transfer and is warned if these do not match. Translations are left until the translation PR is merge.

image

## Types of changes

- Bug fix _(non-breaking change which fixes an issue)_
- New feature _(non-breaking change which adds functionality)_

### CODE REVIEW

**Comment by SuperVK** [comment]
Will this also include creating invoices? I heard from Robin that #304 will be closed without merging.

**Comment by JustSamuel** [comment]
@SuperVK id prefer to do invoice creation in a different PR to reduce scope.

**Comment by CodeNamedRobin** [comment]
@JustSamuel is this ready for review? Also for creating invoices?

**Comment by JustSamuel** [comment]
I have had the beautiful opertunity to test my own code whilst making ALL the intro invoices, I also let the BACPM try it and the first impressions were "yoah dat lijkt wel te werken ofnie", so I think this is fine to merge so that I can go do something else with my life.

@SuperVK please review, I will do the translations when I Get rid of this hangover.

**Comment by SuperVK** [comment]
image
I think the date selection could have some improvement, cause it was not clear for me if I have 28 or 29, or both, or none selected. Either something with an outline of green on the dates that have transactions, or in a text box which dates are selected?

**Comment by JustSamuel** [comment]
@SuperVK 
image

**Comment by SuperVK** [comment]
Nice, I dont think I have much comments right now besides the translations obv

**Comment by JustSamuel** [comment]
> Nice, I dont think I have much comments right now besides the translations obv

translations have been fixed

**Comment by SuperVK** [comment]
image
Missed translation in the toast

image
If not too much trouble, is it possible to make the clear also reset all the other values? like selected point of sales, and how much money is selected, cause that just stays and doesn't clear with the button.

File: apps/dashboard/src/modules/financial/components/invoice/create/InvoiceCreate.vue [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+  await apiService.invoices.createInvoice(request).then(async (invoiceResult: AxiosResponse<InvoiceResponse>) => {
+    toast.add({
+      severity: 'success',
+      summary: t('common.toast.success.success'),
+      detail: t('common.toast.success.invoiceCreated'),
+      life: 3000,
+    });
+
+    const invoice = invoiceResult.data;
+    console.error(invoice.id);
```
Comment (SuperVK) [inline apps/dashboard/src/modules/financial/components/invoice/create/InvoiceCreate.vue]: Still needed?

**Review by SuperVK — APPROVED** [review]
Lekker gewerkt!

**Comment by JustSamuel** [comment]
> image Missed translation in the toast
> 
> image If not too much trouble, is it possible to make the clear also reset all the other values? like selected point of sales, and how much money is selected, cause that just stays and doesn't clear with the button.

kinda works, but you need to reselect the user since the two way binding is not that nice atm. (however this will be quite a difficult fix.)

**Pull request merged**